src/test/java/madalinapopescu/tests/tema6/HerokuForms.java

```diff
@@ -0,0 +1,100 @@
+package madalinapopescu.tests.tema6;
+import madalinapopescu.driver.BrowserManager;
+import madalinapopescu.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
+public class HerokuForms {
+    static ChromeDriver driver = null;
+    public static void main(String[] args) {
+        navigateToHerokuFomePage();
+        try{
+            usernameDetails();
+            passordDetails();
+            commentDetails();
+            fileUploadDetails();
+            checkBoxesDetails();
+            radioButtonsDetail();
+            selectDetails();
+            dropdownDetails();
+            scrollToSubmitButton();
+        }catch (NoSuchElementException e){
+            hiddenImputDetails();
+            FileUtils.takeScreenshot(driver, "hidden-imput");
+            System.out.println("Am intrat in blocul de catch");
+            System.out.println("Nu se poate accesa elementul ascuns");
+        }finally {
+            closeBrowser();
+        }
+    }
+        public static void navigateToHerokuFomePage () {
+            driver = BrowserManager.createChromeDriver();
+            driver.get("https://testpages.herokuapp.com/styled/basic-html-form-test.html");
+            System.out.println("Am deschis Heroku Forms page");
+        }
+        public static void usernameDetails () {
+            WebElement username = driver.findElement(By.name("username"));
+            username.sendKeys("Username-ul meu");
+            System.out.println("Am introdus un usermane");
+        }
+        public static void passordDetails () {
+            WebElement password = driver.findElement(By.name("password"));
+            password.sendKeys("ParolaMea");
+            System.out.println("Am introdus o parola");
+        }
+        public static void commentDetails () {
+            WebElement comment = driver.findElement(By.name("comments"));
+            comment.clear();
+            comment.sendKeys("Mie nu imi place sa comentez");
+            System.out.println("Am introdus un comentariu");
+        }
+        public static void fileUploadDetails () {
+            WebElement fileUpload = driver.findElement(By.name("filename"));
```

Comment:
Can you also try to do the upload too? There are several colleagues that did it already @tincageorgianaalina @curtnazarEliz and others. You can take a look in their PRs or ask for their guidance and if you can't deal with it, let me know. Thanks!

Reply:
in stanga unde ai creat clasele, trebuie sa mai creezi un file, sa scri ceva pe el.  Pe urma iei obiectul si ii trimiti path ul de la acel file care l-ai creat anterior. fileUpload.sendkeys(). exemplu meu : 
WebElement fileUpload = driver.findElement(By.name("filename"));
        fileUpload.sendKeys(System.getProperty("user.dir") + "\\src\\test\\java\\elizCurtnazar\\tests\\tema6\\HerokuFormsFile.txt");

"\\src\\test\\java\\elizCurtnazar\\tests\\tema6\\HerokuFormsFile.txt"); -> acesta e path ul de la file ul meu, tu trebuie sa l adaugi pe al tau.

Reply:
Am facut upload-ul cu poza din printscreen!  Este bine asa sau fac un fisier?

Reply:
Please don't use your absolute local path. The test won't run on other machines. That's what Eliz told you above. Thanks! :)
fileUpload.sendKeys(System.getProperty("user.dir") + "\src\test\java\elizCurtnazar\tests\tema6\HerokuFormsFile.txt");
replace in "\src\test\java\elizCurtnazar\tests\tema6\HerokuFormsFile.txt"); with your path to file and according filename
In this way we are giving a relative path inside our project and it will work on every machine.